en/middleware/micrortps.md

```diff
@@ -231,31 +231,38 @@ Before building `px4_ros_com` you will first need to clone the PX4 Firmware repo
 Then install and setup both ROS2 and ROS environments on your development machine
 and separately clone the `px4_ros_com` repo for both the `master` and `ros1` branches (see [above for more information](#px4_ros_com)).
 
-> **Note** Only the master branch is needed for ROS2 (both are needed to target ROS). 
+> **Note** Only the master branch is needed for ROS2 (both are needed to target ROS).
 
 ### Installing ROS and ROS2 and respective dependencies
 
 In order to install ROS Melodic and ROS2 Bouncy on a Ubuntu 18.04 machine, follow the links below, respectively:
 
 1. [Install ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu)
 2. [Install ROS2 Bouncy](https://index.ros.org/doc/ros2/Linux-Install-Debians/)
-
-Install the following component to ensure that the package properly generates the IDL files:
+3. Install the following component to ensure that the package properly generates the IDL files (only required if one is using the ROS2 Bouncy release):
 
 ```sh
 sudo apt install ros-bouncy-rmw-opensplice-cpp
 ```
 
+> **Note** The requirement for this package should soon be deprecated, as `rosidl_generate_dds_interfaces` CMake tools will be used to generate the required IDL files (instead of only using `rosidl_generate_interfaces`).
+
 The install process should also install the `colcon` build tools, but in case that doesn't happen, you can install the tools manually:
 
 ```sh
 sudo apt install python3-colcon-common-extensions
 ```
 
+4. `setuptools` is also required to be installed (using pip or apt):
```

Comment:
I would do this as:
```
1. *setuptools* must also be installed (using *pip* or *apt*):
```

Generally
- List items numbered as `1.`
- Tools get marked up as italic
- files get marked up in bold
- tool commands use backticks (code style).

Above *pip* refers to the tool so italic, but if you were to do a command, it would be `pip list`

Reply:
Makes sense thanks